test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```diff
@@ -452,6 +504,9 @@ spec:
           apiServer:
             # host.docker.internal is required by kubetest when running on MacOS because of the way ports are proxied.
             certSANs: [localhost, host.docker.internal, "::", "::1", "127.0.0.1", "0.0.0.0"]
+            extraArgs: { v: "2"}
+          scheduler:
+            extraArgs: { v: "2"}
```

Comment:
Do we need them? Or does it default to `0` instead?

Reply:
We need something in extraArgs to make this addressable in the patch - I just set them to 2 which is the recommended default logging level, but I can set it to 0 which is what we should have now by default.